docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+# This docker-compose file simulates a production environment
+
+services:
+  concrexit:
+    container_name: concrexit
+    build:
+      context: .
+      dockerfile: ./Dockerfile
```

Comment:
Don't we maybe want to build in GitHub Actions and pull it from GitHub packages?

Reply:
"# This docker-compose file simulates a production environment"

Reply:
"# This docker-compose file simulates a production environment"